src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, ifMarrid = false, partnerName } = person;
+
+  const partnerSex = sex === 'm'
+    ? 'wife'
+    : 'husbend';
+
+  const personMarrid = ifMarrid
```

Comment:
in your demo i can see that all the people are "not married". You should take a look at the logic inside this variable, may be some variable names are not right.